src/main/java/com/Elessar/database/MetricMonitor.java

```diff
@@ -0,0 +1,56 @@
+package com.Elessar.database;
+
+
+import com.Elessar.app.server.RegisterHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Created by Hans on 1/26/19.
+ */
+public class MetricMonitor {
+    private static final Logger logger = LogManager.getLogger(MetricMonitor.class);
+    private int count;
+    private long totalTime;
+    private long start;
```

Comment:
You might want to use Duration here. The advantage is that it makes it super clear what unit you are using.

Also I would rename getAvgTime() to getAvgTimeMs() to make it clear.